src/sql/workbench/parts/notebook/common/models/notebookUtils.ts

```diff
@@ -125,3 +130,54 @@ export function convertVscodeResourceToFileInSubDirectories(htmlContent: string,
 export function useInProcMarkdown(configurationService: IConfigurationService): boolean {
 	return configurationService.getValue('notebook.useInProcMarkdown');
 }
+
+export function getClusterEndpoints(serverInfo: ServerInfo): IEndpoint[] | undefined {
```

Comment:
This code was copied from the extension side, we really need a way to share it (or just move the code there...)